src/bramble/client/main.js

```diff
@@ -506,11 +510,19 @@ define([
 
             // With successful fs operations that create, update, delete, or rename files,
             // we also trigger events on the bramble instance.
-            function genericFileEventFn(type, filename, callback) {
+            function genericFileEventFn(type, filename,callback,oldSize) {
```

Comment:
We can undo all these changes, since we'll do it internally on the `_fs` ourselves.